### ISSUE TITLE
IOS build issue - dependencies import issue

### ISSUE THREAD

**Issue by kokulansk** [dpa99c/cordova-plugin-firebasex#169]
# Bug report

**Current behavior:**

**Expected behavior:**

**Steps to reproduce:**

**Screenshots**

**Environment information**

- Cordova CLI version 
	- `cordova -v`
- Cordova platform version
	- `cordova platform ls`
- Plugins & versions installed in project (including this plugin)
    - `cordova plugin ls`
- Dev machine OS and version, e.g.
    - OSX
        - `sw_vers`
    - Windows 10
        - `winver`
        
_Runtime issue_
- Device details
    - _e.g. iPhone X, Samsung Galaxy S8, iPhone X Simulator, Pixel XL Emulator_
- OS details
    - _e.g. iOS 12.2, Android 9.0_	
	
_Android build issue:_	
- Node JS version
    - `node -v`
- Gradle version
	- `ls platforms/android/.gradle`
- Target Android SDK version
	- `android:targetSdkVersion` in `AndroidManifest.xml`
- Android SDK details
	- `sdkmanager --list | sed -e '/Available Packages/q'`
	
_iOS build issue:_
- Node JS version
    - `node -v`
- XCode version

**Related code:**
```
insert any relevant code here such as plugin API calls / input parameters
```

**Console output**
<details>
<summary>console output</summary>

```

// Paste any relevant JS/native console output here

```

</details><br/><br/>

**Other information:**

![Screenshot 2019-10-03 at 13 28 06](https://user-images.githubusercontent.com/8098810/66177327-b1840d80-e67e-11e9-8453-9580593e4a86.png)

After fixing pod path getting this error
![Screenshot 2019-10-03 at 13 44 07](https://user-images.githubusercontent.com/8098810/66177396-eabc7d80-e67e-11e9-8a26-aa98097f2503.png)

**Comment by dpa99c** [comment]
Please ensure you have updated Cocoapods as [outlined in the documentation](https://github.com/dpa99c/cordova-plugin-firebasex#cocoapods).

Please try building the [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) to rule out other possible causes in your own project.

**Comment by kokulansk** [comment]
I tried the example project it's giving the same error for me 
![Screenshot 2019-10-04 at 07 50 24](https://user-images.githubusercontent.com/8098810/66176560-d1660200-e67b-11e9-9654-aa00b3f265ac.png)

**Comment by distinctdan** [comment]
I just had the same issue. Someone on the cocoapods repo said this plugin needs to be updated to use the newer cocoapods style where it doesn't automatically sync the whole repo. Here's the issue I filed over there: https://github.com/CocoaPods/CocoaPods/issues/9228

I worked around this by uninstalling cocoapods, installing version 1.7.5, syncing the repo, then upgrading to 1.8.3.

**Comment by dpa99c** [comment]
This should be resolved by #173
